File: Generador/PersonaNaturalEntity.java
package EntityLayer;

import com.fasterxml.jackson.annotation.JsonProperty;
import Enumerados.ProcessActionEnum;
import java.util.Date;

public class PersonaNaturalEntity {

    @JsonProperty("PersonaNaturalId")
    private int personaNaturalId;
    public int getPersonaNaturalId() {
        return personaNaturalId;
    }

    public void setPersonaNaturalId(int personaNaturalId) {
        this.personaNaturalId = personaNaturalId;
    }

    @JsonProperty("TipoDocumentoIdentidadId")
    private int tipoDocumentoIdentidadId;
    public int getTipoDocumentoIdentidadId() {
        return tipoDocumentoIdentidadId;
    }

    public void setTipoDocumentoIdentidadId(int tipoDocumentoIdentidadId) {
        this.tipoDocumentoIdentidadId = tipoDocumentoIdentidadId;
    }

    @JsonProperty("NumDocumento")
    private String numDocumento;
    public String getNumDocumento() {
        return numDocumento;
    }

    public void setNumDocumento(String numDocumento) {
        this.numDocumento = numDocumento;
    }

    @JsonProperty("Nombres")
    private String nombres;
    public String getNombres() {
        return nombres;
    }

    public void setNombres(String nombres) {
        this.nombres = nombres;
    }

    @JsonProperty("ApellidoPaterno")
    private String apellidoPaterno;
    public String getApellidoPaterno() {
        return apellidoPaterno;
    }

    public void setApellidoPaterno(String apellidoPaterno) {
        this.apellidoPaterno = apellidoPaterno;
    }

    @JsonProperty("ApellidoMaterno")
    private String apellidoMaterno;
    public String getApellidoMaterno() {
        return apellidoMaterno;
    }

    public void setApellidoMaterno(String apellidoMaterno) {
        this.apellidoMaterno = apellidoMaterno;
    }

    @JsonProperty("FechaNacimiento")
    private Date fechaNacimiento;
    public Date getFechaNacimiento() {
        return fechaNacimiento;
    }

    public void setFechaNacimiento(Date fechaNacimiento) {
        this.fechaNacimiento = fechaNacimiento;
    }

    @JsonProperty("UbigeoId")
    private int ubigeoId;
    public int getUbigeoId() {
        return ubigeoId;
    }

    public void setUbigeoId(int ubigeoId) {
        this.ubigeoId = ubigeoId;
    }

    @JsonProperty("Direccion")
    private String direccion;
    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    @JsonProperty("Telefono")
    private String telefono;
    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    @JsonProperty("Correo")
    private String correo;
    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    @JsonProperty("Genero")
    private int genero;
    public int getGenero() {
        return genero;
    }

    public void setGenero(int genero) {
        this.genero = genero;
    }

    @JsonProperty("EstadoCivil")
    private int estadoCivil;
    public int getEstadoCivil() {
        return estadoCivil;
    }

    public void setEstadoCivil(int estadoCivil) {
        this.estadoCivil = estadoCivil;
    }

    @JsonProperty("Action")
    private ProcessActionEnum action = ProcessActionEnum.Loaded;

    public int getAction() {
        return action.getValor();
    }

    public void setAction(ProcessActionEnum Action) {
        this.action = Action;
    }

}

File: SharedApi/Space/Space.EntityLayer/PersonaNaturalEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Space.EntityLayer
{
    public partial class PersonaNaturalEntity : BaseEntityObject
    {

        public PersonaNaturalEntity()
        {
            PersonaNaturalId = GetInt32Default();
            Nombres = GetStringDefault();
            ApellidoPaterno = GetStringDefault();
            ApellidoMaterno = GetStringDefault();
            FechaNacimiento = DateTime.Now;
            Direccion = GetStringDefault();
            Telefono = GetStringDefault();
            Correo = GetStringDefault();
            SexoId = GetInt32Default();
            EstadoCivilId = GetInt32Default();
            TipoDocumentoIdentidadId = GetInt32Default();
            NumDocumento = GetStringDefault();
            UbigeoId = GetInt32Default();
            FechaRegistro = DateTime.Now;
            CodUsuario = GetStringDefault();
        }

        public Int32 PersonaNaturalId { get; set; }
        public Int32 TipoDocumentoIdentidadId { get; set; }
        public String NumDocumento { get; set; }
        public String Nombres { get; set; }
        public String ApellidoPaterno { get; set; }
        public String ApellidoMaterno { get; set; }
        public DateTime FechaNacimiento { get; set; }
        public String Direccion { get; set; }
        public String Telefono { get; set; }
        public String Correo { get; set; }
        public Int32 SexoId { get; set; }
        public Int32 EstadoCivilId { get; set; }
        public Int32 UbigeoId { get; set; }
        public String CodUsuario { get; set; }
        public DateTime FechaRegistro { get; set; }

    }
}
